python/lsst/obs/base/cli/opt/arguments.py

```diff
@@ -19,8 +19,4 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-from lsst.daf.butler.cli.utils import MWArgumentDecorator
-
-instrument_argument = MWArgumentDecorator(
-    "instrument", help="The name or fully-qualified class name of an instrument."
-)
+"""Shared arguments used by obs_base command-line tools."""
```

Comment:
If this file is empty should it be removed?

Reply:
I was wondering about that and whether it was better to keep it around to show where arguments should go or to remove it completely. I suppose I'll remove it.